sympy/matrices/expressions/determinant.py

```diff
@@ -32,14 +33,16 @@ def __new__(cls, mat):
     def arg(self):
         return self.args[0]
 
-    def doit(self, expand=False):
+    def doit(self, expand=False, **kwargs):
```

Comment:
Also, is `expand` used at all?

Reply:
@gxyd ,
I am very sorry if I have disturbed the mentors. I didn't mean to do so. 
And I didn't know that we do not generally ping mentors on the PR description.
Apologies.

I'll check on your reviews and update.

Reply:
In this case, I do agree that you have nothing to expand, since det will give you a value. Normal function definition of `doit` specifies an `expand`, so it was there. I think it can be removed.

Reply:
@asmeurer I think we should remove the `expand=False`? Also, I don't see any other class having `expand` in its API either.